src/index.js

```diff
@@ -1,10 +1,38 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+let myElement = document.getElementById("biologis-json-converter-helper");
+if (myElement === null) {
+  let newElement = document.createElement("div");
+  newElement.setAttribute("id", "biologis-json-converter-helper");
+  document.body.appendChild(newElement);
+}
+
+function changeInputText(inputEl, valuesString) {
+  inputEl.value = valuesString;
+  document.getElementById("biologis-json-converter-helper").style.display =
```

Comment:
You could use `myElement` here now.
Also myElement is not a good name ;)

Reply:
I will change the name, just wanted you to review first to see if it's okay.